### DISCUSSION TITLE
Database Import Row Size Too Large

### DISCUSSION THREAD

**Discussion by markosege** [wardenenv/warden#804] [Help & Support]
### Version of Warden

0.14.3

### Operating System and Installation Method

Mac M3 via brew

### Describe the Bug

While importing db dump from Aurora RDS keep getting this error any help is appreciated
```

ERROR 1118 (42000) at line 11059: Row size too large (> 8126). Changing some columns to TEXT or BLOB may help. In current row format, BLOB prefix of 0 bytes is stored inline.                                   ]  11% ETA 1:27:21
ERROR: Command `"$WARDEN_BIN" env exec -T db mysql -u"${MYSQL_USER}" -p"${MYSQL_PASSWORD}" --database="${MYSQL_DATABASE}" "${WARDEN_PARAMS[@]:1}" "$@"` at /opt/homebrew/Cellar/warden/0.14.3/commands/db.cmd:31 failed with exit code 1
```

### To Reproduce

pv /path/to/dump.sql.gz | gunzip -c | warden db import

### Expected Behavior

db should be imported

**Comment by iranimij** [comment]
I also get the same error, Did you find the solution?

**Comment by iranimij** [comment]
After removing the old tables, and trying to import again, It works again.
